### ISSUE TITLE
whether to shuffle the data or not

### ISSUE THREAD

**Issue by sgbaird** [sparks-baird/xtal2png#84]
Either in `mp-time-split` (when using `.get_train_and_val_data()` etc., of course only shuffling after splitting into train and val) or in the generative model. Might cause issues if really similar compounds are in the same batch.

- https://datascience.stackexchange.com/questions/24511/why-should-the-data-be-shuffled-for-machine-learning-tasks
- https://stats.stackexchange.com/questions/245502/why-should-we-shuffle-data-while-training-a-neural-network

Probably better to just shuffle the data by default rather than include it as a tunable hyperparameter during hyperparameter optimization. Might leave as a kwarg with default set to True. Of course, if the generative model uses data from a directory without shuffling that data, it could be problematic. Will check on `denoising_diffusion_pytorch` to see if the data gets shuffled by default.

**Comment by sgbaird** [comment]
It seems like the data is getting shuffled by default in `denoising_diffusion_pytorch`:

https://github.com/lucidrains/denoising-diffusion-pytorch/blob/79c5f045e10461c3f56f2ecf3767f318ec3719e9/denoising_diffusion_pytorch/denoising_diffusion_pytorch.py#L601

https://github.com/lucidrains/denoising-diffusion-pytorch/issues/37

**Comment by sgbaird** [comment]
Add note in mp-time-split and leave as-is
